syntax = "proto3";

package chat;

// The chat message request.
message ChatRequest {
  string user = 1;
  string message = 2;
}

// The chat message reply.
message ChatReply {
  string message = 1;
}

// The chat service definition.
service ChatService {
  // BiDi streaming RPC.
  rpc SendMessage(ChatRequest) returns (ChatReply) {}
}

